features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -60,11 +62,13 @@ internal class RumViewManagerScope(
             startForegroundView(event, writer)
             lastStoppedViewTime?.let {
                 val gap = event.eventTime.nanoTime - it.nanoTime
-                sdkCore.internalLogger.log(
-                    InternalLogger.Level.INFO,
-                    listOf(InternalLogger.Target.TELEMETRY, InternalLogger.Target.MAINTAINER),
-                    { MESSAGE_GAP_BETWEEN_VIEWS.format(Locale.US, gap) }
-                )
+                if (gap in 1 until THREE_SECONDS_GAP) {
```

Comment:
according to the telemetry, we can also have a negative gap by some reason. Do we want to log negative gaps as well? This is something abnormal imo.

Reply:
True, I'll add a specific metric for that case

Reply:
seems like it will be in another PR? this PR was merged without this update